orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -626,7 +723,8 @@ Status ToPropertyBag(const onnxruntime::fbs::PropertyBag& fbs_property_bag,
  * @return Status of the operation.
  */
 Status ToModelProto(gsl::span<const uint8_t> checkpoint_bytes,
-                    ONNX_NAMESPACE::ModelProto& model_proto) {
+                    ONNX_NAMESPACE::ModelProto& model_proto,
+                    const PathString& checkpoint_path) {
```

Comment:
Should this be an optional input? Please change the documentation here as well.

Reply:
I don't think it makes sense to have it as an optional input since the method that passes in the checkpoint_path is LoadCheckpointToModel which gets the checkpoint_path from the API call. Updated the doc 👍